<launch>
    <node pkg="gremsy_base" type="GimbalNode" name="gremsy_base" output="screen">
        <rosparam command="load" file="$(find gremsy_base)/config/config.yaml"/>
    </node>
</launch>
